content/blog/jt15s/2020-12-31_hello_from_the_haiku_promotion_team.md

```diff
@@ -0,0 +1,74 @@
+
+### Partnerships with other Haiku-related sources
+If you haven't heard of them already, [BeSly](https://www.besly.de/index.php/de/) and [Haikunauts](https://haikunauts.home.blog/) are the two major third party Haiku sources. BeSly is a wiki with tips and tricks on using Haiku, whilst Haikunauts is a blog aimed at promoting Haiku as an OS. As mentioned above, we will work in partnership with these two sources and see what we can do together to help spread the word about Haiku.
+
+Additionally, we are also eager to work with any smaller blogs, websites or channels. If you run a blog, website, channel, or any other source, please contact me via the forums and let me know!
+
+### General
+I am thinking of running an advertisement in [Read the Docs](https://readthedocs.org/), a documentation website that serves a wide array of open-source projects. Read the Docs [allocates a portion of their advertising inventory to FOSS projects](https://docs.readthedocs.io/en/stable/advertising/ethical-advertising.html#community-ads) such as ourselves, hence this is a good opportunity to promote Haiku. What will go on the advertisement has not been decided as of now.
+
+scott_puopolo18 is working on a set of interview questions for our developers to give the public a glimpse into the world of Haiku development, as well as shining a light on the challenges and opportunities in an open-source project. The predetermined set of questions means that we don't have to write new questions for each developer. To save time, we can simply send them off and the developer can write their response.
+
+We are also planning to distribute more content via our often under-utilised social media platforms, as well as update them with current information (i.e. adding profile page art, applying for verification, updating or adding links etc.). Short-form posts on social media are what immediately comes to mind, although we are considering live-streams or videos if there is someone willing to help out.
+
+Haikunauts also has several other ideas for promotional events, but we will share these with you later as we are still working out the details of these events.
+
+### In closing
+We are super excited to see how we can promote Haiku and help the project gain new members and volunteers. **[Remember, if you’d like to join, don’t hesitate and send me a private message on the forums!](https://discuss.haiku-os.org/u/jt15s)**
+
+I’d also like to take this opportunity to wish everyone a very Happy New Year - here’s to a more fruitful, happy and lucky new year!
+
+Finally, I'd like to thank pulkomany and waddlesplash who reviewed this post on Github and gave me plenty of constructive feedback on how I could improve this.
```

Comment:
PulkoMandy (you're missing a d and the uppercase letters are important :) )

Reply:
Apologies - I must have been typing this when I was tired :)